gadget/layout_test.go

```diff
@@ -925,31 +925,6 @@ volumes:
 	})
 }
 
-func (p *layoutTestSuite) TestLayoutVolumeOffsetWriteBadRelativeTo(c *C) {
```

Comment:
why is this test not relevant anymore? because it cannot be written is passing things in explicitly?